pkg/sawmill/sawmill.go

```diff
@@ -0,0 +1,87 @@
+package sawmill
+
+// If you're wondering why this package is named sawmill, I do too: https://clips.twitch.tv/PlacidOutstandingPelicanCharlieBitMe
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"cloud.google.com/go/logging"
+	"google.golang.org/api/option"
+	logpb "google.golang.org/genproto/googleapis/logging/v2"
+)
+
+// Logger holds the newly generated error logging client, so that it can be used for our own, more simple custom logging functions instead of the ones the Google provides
+type Logger struct {
+	Logger *logging.Logger // The Logger
+
+	InitError bool // If their was an error with the initialization, it won't log to the cloud
+
+	ServiceName string // The service that logging was created for
+}
+
+// InitClient creates a logging client
+func InitClient(projectID string, credentials string, environment string, serviceName string) (Logger, error) {
```

Comment:
What does credentials mean here? Thinking if we can change this property name a tad, may be easier to understand

Reply:
The credentials are the raw JSON pulled from the env. I'll change it to something like `credentialsJSON`.

Reply:
Still needs update!

Reply:
Weird, swear I did that

Reply:
Hey my bad Github's PR system is weird... Looks like you did. Thanks!